tests/fixtures/vault.py

```diff
@@ -30,7 +31,7 @@
 VAULT_URL = DEFAULT_TEST_CONFIG.vault_url
 DEFAULT_PORT = 8200
 VAULT_PATH = DEFAULT_TEST_CONFIG.vault_path
-VAULT_TOKEN = DEFAULT_TEST_CONFIG.vault_token
+VAULT_TOKEN = "dev-token"
```

Comment:
Where is this token now used?

Reply:
It is used to configure vault like initiation with the desired state, like changing the default mount point for testing.

Reply:
What @dontseyit said